packages/core/src/command/hl7v2-subscriptions/__tests__/hl7v2-roster-generator.test.ts

```
@@ -20,54 +68,62 @@ describe("AdtRosterGenerator", () => {
-      city: "Cambridge",
-      state: USState.MA,
-      zip: "02139",
-      country: "USA",
-    };
-
-    const ssn = "123-45-6789";
-    const driversLicense = "DL123456";
-    const phoneNumber = "600-700-8000";
-    const email = "john@doe.com";
+      });
+      delete patient.data.contact;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)